sphinx/builders/linkcheck.py

```diff
@@ -134,6 +136,10 @@ def check_uri():
             # split off anchor
             if '#' in uri:
                 req_url, anchor = uri.split('#', 1)
+                for rex in self.anchors_ignore:
+                    if rex.match(uri):
```

Comment:
So there was a question about this piece before as to whether it should use 'anchor' instead of 'uri' in the PR #2389.

I think it depends on whether matching is desirable on being able to skip checking anchors on entire sites, or whether would have to list all anchors to skip checking on, and the odds that you could have the same anchor name appearing for two sites where you'd like to check one and skip the other.

Happy to change, just thought I'd ask the question with better explanation of the original thoughts first.